feature/bgp/multipath/otg_tests/bgp_multipath_wecmp_lbw_community_test/README.md

```diff
@@ -19,43 +19,117 @@ Validate BGP in multipath UCMP support with link bandwidth community
     *   ATE port-2 and DUT port-2
     *   ATE port-3 and DUT port-3
 *   Enable an Accept-route all import-policy/export-policy for eBGP session
-    under the neighbor AFI/SAFI
-*   Create an IPv4 internal target network attached to ATE port 2 and 3
+    under the neighbor AFI/SAFI - IPv6 unicast and IPv4 unicast.
+*   Create an single IPv4 internal target network attached to ATE port 2 and 3
+*   [TODO] Create an single IPv6 internal target network attached to ATE port 2 and 3
+
 
 ### Tests
 
-*   RT-1.52.1: Verify use of community type
+*   RT-1.52.1: Verify use of unequal community type
+
+    *   Configure ATE port 1, 2 and 3 on different AS, with boths AFI/SAFI
```

Comment:
I recommend breaking this into two sections:
* Config generation + gnmi.Set REPLACE
* Validate (traffic generate and check counters) section.

Then you can more clearly refer to the config and 'reuse'  it in the following subtests